src/Entity/MetatagContext.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\metatag\Entity\MetatagContext.
+ */
+
+namespace Drupal\metatag\Entity;
+
+use Drupal\Core\Config\Entity\ConfigEntityBase;
+use Drupal\metatag\MetatagContextInterface;
+
+/**
+ * Defines the Metatag context entity.
+ *
+ * @ConfigEntityType(
+ *   id = "metatag_context",
+ *   label = @Translation("Metatag context"),
+ *   handlers = {
+ *     "list_builder" = "Drupal\metatag\MetatagContextListBuilder",
+ *     "form" = {
+ *       "add" = "Drupal\metatag\Form\MetatagContextForm",
+ *       "edit" = "Drupal\metatag\Form\MetatagContextForm",
+ *       "delete" = "Drupal\metatag\Form\MetatagContextDeleteForm"
+ *     }
+ *   },
+ *   config_prefix = "metatag_context",
+ *   admin_permission = "administer site configuration",
+ *   entity_keys = {
+ *     "id" = "id",
+ *     "label" = "label",
+ *     "uuid" = "uuid"
+ *   },
+ *   links = {
+ *     "canonical" = "/admin/structure/metatag_context/{metatag_context}",
+ *     "edit-form" = "/admin/structure/metatag_context/{metatag_context}/edit",
+ *     "delete-form" = "/admin/structure/metatag_context/{metatag_context}/delete",
+ *     "collection" = "/admin/structure/visibility_group"
+ *   }
+ * )
+ */
+class MetatagContext extends ConfigEntityBase implements MetatagContextInterface {
```

Comment:
Once we move the defaults out of it, then we will only have the id, the label and the tags sequence that points to a ste of Tag config entities. Do you think that we should still change it or leave it as it is?